portx/modules/ROOT/pages/availability.adoc

```diff
@@ -2,10 +2,14 @@
 
 You can configure windows (periods of time) during which partners or transactions are on-hold or inactive.
 
-include::{partialsdir}/edit-settings.adoc[]
+include::{partialsdir}/relationships-settings.adoc[]
 [start=3]
 
-. In the *CONFIGURATION* section of the left-hand navigation pane on the <<partner-configuration.adoc#img-company-information, Company Information Page>>, click *Availability*.
+. Select a relationship with partners and/or applications, from the list. +
+Alternatively, you can use the Filter Bar to search for a relationship. 
```

Comment:
Cap Relationship

Reply:
This change is now applied to all applicable configuration docs.